src/tribol/utils/ContactPlaneOutput.cpp

```diff
@@ -98,6 +98,7 @@ void WriteContactPlaneMeshToVtk( const std::string& dir, const VisType v_type,
             overlaps = true;
```

Comment:
I think the default rank should be 0 (see L65)?

Reply:
This ought to just be reset from invalid to valid? Minor detail.